scripts/release_processes/firecloud-develop.dot

```diff
@@ -2,40 +2,71 @@ digraph {
 
   # Nodes
 
-  release_cromwell [shape=oval label="Release new Cromwell version! Woohoo!"];
+  subgraph cluster_0 {
+    label="\lPublishing a RELEASE"
+    style=filled
+    node [style=filled,color=white];
+    color="lightgrey"
 
-  fcdev_branch [shape=oval label="Make firecloud-develop PR for new Cromwell version"];
-  fcdev_test [shape=oval label="Retest firecloud-develop PR. Confirm Cromwell version in Jenkins console logs"];
-  fcdev_update_dev [shape=oval label="Checkout 'dev' branch of firecloud-develop and 'git pull' the latest changes"];
-  fcdev_final_swatomation [shape=oval label="Retest firecloud-develop PR. Confirm Cromwell version in Jenkins console logs"];
-  fcdev_merge [shape=oval label="Merge firecloud-develop PR"];
+    release_cromwell [shape=oval label="PUBLISH: We have a new Cromwell version! Woohoo!"];
+  }
 
-  dspjenkins_PR [shape=oval label="Make dsp-jenkins PR setting new Cromwell version"];
-  dspjenkins_merge [shape=oval label="Merge dsp-jenkins PR"];
+  subgraph cluster_1 {
+    label="\lPublishing a HOTFIX"
+    style=filled
+    node [style=filled,color=white];
+    color="lightgrey"
 
-  jenkins_set [shape=oval label="[Jenkins] Use dsl-seed to make our dsp-jenkins branch the default"];
-  jenkins_reset [shape=oval label="[Jenkins] Use dsl-seed to make 'master' the default again"];
+    hotfix_changes [shape=oval label="HOTFIX: Changes get merged to a hotfix branch"];
+    publish_hotfix [shape=oval label="Travis will automatically publish to dockerhub"];
+  }
 
-  qa_perf [shape=oval label="QA performance testing"];
+  subgraph cluster_2 {
+    label="Release\ninto Terra"
+    style=filled
+    node [style=filled,color=white];
+    color="lightgrey"
+
+    fcdev_pr [shape=oval label="Make (or update) the firecloud-develop PR for the new Cromwell version"];
+    confirm_cromwell_v [shape=oval label="[Jenkins] Check for the correct Cromwell version hash in fiab-start logs\n(otherwise re-run 'dsl-seed' and use 'jenkins retest')"];
+    confirm_cromwell_v [shape=oval label="[Jenkins] Check for the correct Cromwell version hash in fiab-start logs\n(otherwise re-run 'dsl-seed' and use 'jenkins retest')"];
+    fcdev_success [shape=oval label="Wait for swatomation to succeed"];
+
+    fcdev_merge [shape=oval label="Merge firecloud-develop PR"];
+
+    dspjenkins_PR [shape=oval label="Make dsp-jenkins PR setting new Cromwell version"];
+    dspjenkins_merge [shape=oval label="Merge dsp-jenkins PR"];
+
+    qa_perf [shape=oval label="QA performance testing (skip for hotfix releases)"];
```

Comment:
Suggestion:
`(for hotfix releases, this step is not blocking, but these tests can be run on the side)`